spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/AetherModuleResolver.java

```diff
@@ -162,4 +147,59 @@ public void serviceCreationFailed(Class<?> type, Class<?> impl, Throwable except
 		});
 		return locator.getService(RepositorySystem.class);
 	}
+
+	@Override
+	public Resource[] resolve(Coordinates root, Coordinates[] includes, CoordinatesFilter[] excludes) {
+		Assert.hasText(root.getGroupId(), "'groupId' cannot be blank.");
```

Comment:
Ah, correct. For some reason I was under the impression that the includes did not require the same props as the root, but they do.